.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Build and release Unreal Plugin  🎉 
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  packages: write
+
+env:
+  CPP_SDK_TAG: "2.4.0"
```

Comment:
This can't be right. I think you want to merge into branch 1.0 and then CPP_SDK_TAG must be 2.3 and UE versions below should not include 5.1.